security-demo/cloud-stub/pkg/controller/cloudhub/stub.go

```diff
@@ -215,7 +215,7 @@ func (tm *StubCloudHub) Start() {
 func (tm *StubCloudHub) PlacementServer() {
 	fmt.Println("started placement server")
 	router := mux.NewRouter()
-	router.HandleFunc("/pod", tm.podHandler)                                       // for pod test
+	//router.HandleFunc("/pod", tm.podHandler)                                       // for pod test
```

Comment:
remove the dead line.

Reply:
Oops!! Accidental test code checked-in. Reverted back the changes.
Thanks for pointing out.